src/components/victory-chart/victory-chart.js

```diff
@@ -307,6 +315,7 @@ export default class VictoryChart extends React.Component {
         ref: index,
         key: index,
         standalone: false,
+        theme: props.theme,
```

Comment:
Then something else is going wrong. `theme` is added to `props` either by being provided by the user or by being added by `defaultProps`, so having `defaultProps` is circular / nonsensical.